x-pack/plugins/alerts/server/alerts_client.test.ts

```diff
@@ -4086,31 +4086,33 @@ describe('update()', () => {
       taskManager.runNow.mockReset();
       taskManager.runNow.mockReturnValue(resolveAfterAlertUpdatedCompletes);
 
-      await alertsClient.update({
-        id: alertId,
-        data: {
-          schedule: { interval: '10s' },
-          name: 'abc',
-          tags: ['foo'],
-          params: {
-            bar: true,
-          },
-          throttle: null,
-          actions: [
-            {
-              group: 'default',
-              id: '1',
-              params: {
-                foo: true,
-              },
+      alertsClient
```

Comment:
Why convert this away from async/await? Errors thrown in the then-back will no longer bubble up to the test appropriately with this change.